workflows/methylation/methylation-cohort.wdl

```diff
@@ -0,0 +1,245 @@
+version 1.1
+
+workflow methylation_cohort {
+    meta {
+        description: "Process methylation data for a cohort of samples"
+        outputs: {
+            combined_beta: "Matrix (in CSV format) containing beta values for every (common) probe on the array as rows and all of the input samples as columns.",
+            filtered_beta: "Matrix (in CSV format) containing only beta values for the retained probes (top N highest standard deviation) for all provided samples.",
```

Comment:
So the difference between these files is that `combined_beta` contains _every common_ probe, and `filtered_beta` is that same set but filtered to top N hight STDEV? Can that difference be stated explicitly?

Reply:
Yes, that is the difference between the files. Where would you want that to be stated? It is in the quotes `parameter_meta` entries. Do those need additional text to make clear?

Reply:
I guess it's fine. I'm not sure where to add that clarifying text... I understood correctly after all.